### ISSUE TITLE
Does not create collection via `init_beanie` function call when there are no user-defined index!

### ISSUE THREAD

**Issue by harshithjv** [roman-right/beanie#551]
There was same issue reported here: https://github.com/roman-right/beanie/issues/315

If the `Document` class' field does not have `Indexed` option or `Settings.indexes` is not defined, then collection is not created on Mongo DB as empty document cannot created in MongoDB unless there is atleast one `index` is defined. 
- This is confusing as default field `_id` should have an index, right? Why it's skipping creating collection that does not have user-defined index?
- We can create collection in Mongo CLI without specifying index, right?

Why index mandatory in `beanie` when not mandatory in `motor` or `mongodb`?

**Comment by roman-right** [comment]
Hi!
I didn't get what you mean by collection creating. This is a lazy operation in MongoDB. If you apply any operation against a collection it creates there. Operations could be index creation and document insertion. Or you are asking about something else?

**Comment by harshithjv** [comment]
@roman-right 

Looks I'd not called `init_beanie()` before start of any operation on models via python prompts. As result I was seeing `CollectionWasNotInitialized` exception and I thought it was because `beanie` was not creating models that does not have indexes. Since I started working on this library yesterday it's very frustrating experience.

Like in other link I tagged on main comment, many newbies to your library might face this issue. It would be nice to have this issue noted in separate box to help the beginners like us!

Thanks for the response. I'm closing this as resolved issue. Cheers.😀👍
